spring-integration-gcp/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>spring-cloud-gcp</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<name>Spring Integration GCP</name>
+	<description>Integration modules for Google Cloud Platform</description>
+	<artifactId>spring-integration-gcp</artifactId>
+	<organization>
```

Comment:
As I said, IANAMExpert, but I would have thought that as long as it's not mentioned in the parent pom, maven will ignore it; it can have its own stand-alone pom.

Bamboo allows you to specify the working directory - if it can't be done with Jenkins (all the other SI modules are built with Bamboo anyway, so the build probably belongs there anyway).

In any case, I am ok with deferring resolution on this.